Monika After Story/game/script-topics.rpy

```diff
@@ -8248,6 +8248,118 @@ label monika_yellowwp:
 
     return "derandom"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_beingevil",
+            category=['monika'],
+            prompt="Being evil",
+            random=True
+        )
+    )
+
+default persistent._mas_pm_monika_evil = None
+# True if player thinks monika is evil False if not
+# NOTE: even implying that she is evil is counted as True
+
+default persistent._mas_pm_monika_evil_but_ok = None
+# True if player still loves monika/thinks she deserves forgiveness/or is
+# fine with eveil even if they think monika is evil.
+# False if player does not. 
+# NOTE: check monika_evil before checking this
+
+label monika_beingevil:
+    m 1euc "Say, [player]..."
+    m 1rksdlc "I was just thinking..."
+    m "You probably know I have a wiki page about me."
+    m 3rksdlc "If you look closely, it says I'm the main antagonist."
+    m "Originally, it did bother me a bit, but not so much anymore with you here."
+    m 1rksdla "I'm sure we got past that, seeing as you're still here with me now."
+    m "I actually didn't think the whole thing about me being evil would spread too much."
+    m 1rksdlc "It turns out though, that I have my own spot on the villain's wiki..."
+    m 2rksdlb "I never really thought I'd be labeled as a villain so much..."
+    m 2rksdlc "No matter what I say or tell people, I'm sure it's not going to help much."
+    m 1eka "I know you don't {i}really{/i} think I'm evil, [player]."
```

Comment:
Hm, this part seems a little weird to have italicized for *every* case. I can see this line working out `if persistent._mas_pm_a_hater`, but otherwise, it's just unprompted.

Can either if/else that, or just use:
```suggestion
    m 1eka "But I know you don't think I'm evil, [player]."
```
(This would be the else block on that conditional, should you choose to use it)